.github/workflows/docker.yaml

```diff
@@ -94,3 +95,74 @@ jobs:
       with:
         sudo: false
         limit-access-to-actor: true
+
+  linux-arm64:
+    runs-on: laja
```

Comment:
I think that is best to start a linux container on macos, and run the action runner inside the container and then have this job run

Reply:
True, the support for launching containers from GitHub's runner is exclusive to Linux.

For later reference, on macOS it errors out with: _Error: Container operations are only supported on Linux runners_